src/NUnitEngine/Addins/vs-project-loader/VSProject.cs

```diff
@@ -318,24 +313,22 @@ private void LoadLegacyCppProject()
             foreach (XmlNode configNode in _doc.SelectNodes("/VisualStudioProject/Configurations/Configuration"))
             {
                 string name = RequiredAttributeValue(configNode, "Name");
-                int config_type = System.Convert.ToInt32(RequiredAttributeValue(configNode, "ConfigurationType"));
-                string dirName = name;
-                int bar = dirName.IndexOf('|');
+                int config_type = Convert.ToInt32(RequiredAttributeValue(configNode, "ConfigurationType"));
+                var dirName = name;
```

Comment:
Great example of when var should _not_ be used, IMO. There's nothing in the line that tells the reader what name is. I have to look up two lines to understand  that. Now if the type name were something in the order of 15 characters, I might go the other way. But typing or reading "string" isn't that hard.

My point isn't that my view is correct, it's that there's judgement involved.